runtime/compiler/optimizer/EscapeAnalysis.cpp

```diff
@@ -1169,6 +1187,16 @@ int32_t TR_EscapeAnalysis::performAnalysisOnce()
                rememoize(candidate);
                _candidates.remove(candidate);
                }
+            else
+               {
+               TR_OpaqueClassBlock *clazz = (TR_OpaqueClassBlock*)classNode->getSymbol()->castToStaticSymbol()->getStaticAddress();
+
+               if (TR::Compiler->cls.isValueTypeClass(clazz))
+                  {
+                  if (trace())
+                     traceMsg(comp(), "   Fail [%p] because array has value type elements\n", candidate->_node);
```

Comment:
How does this fail happen - the trace says there is a failure, but the candidate isn't removed?

Reply:
Thanks for catching that mistake.

Reply:
Corrected in commit https://github.com/eclipse/openj9/pull/9139/commits/d897b437a02626a45d5be525d10a50d192db14c1